easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.20_fix-cpuid-neoverse-v1-n2.patch

```diff
@@ -0,0 +1,15 @@
+# This fixes an incorrect autodetection of NeoverseV1 architecture at build time
+# Fixes https://github.com/EESSI/software-layer/pull/310#issuecomment-1734345487
```

Comment:
@casparvl Let's mention that this was extracted from https://github.com/OpenMathLib/OpenBLAS/commit/09b8545fc51316d0fecf34c9e753b8a20358a3e8, which is part of https://github.com/OpenMathLib/OpenBLAS/pull/3586 (just so we know this fix is included in OpenBLAS v0.3.21 and newer, albeit a bit sneakily)

Reply:
Done